### ISSUE TITLE
rpm install block file will load brd modules automatically

### ISSUE THREAD

**Issue by JetXujing** [rpm-software-management/rpm#2275]
I have a spec as follows:
[root@localhost SPECS]# cat test.spec
```
Name: test
Version: 1.0
Release: 1
Group: Testing
License: GPL
Summary: Simple rpm demonstration.

%description
Simple rpm demonstration.

%prep

%build

%install
mknod %{buildroot}/test-block b 1 2

%files
/test-block
```

1. `rpmbuild -bb test.spec`
2. `rmmod brd`
3. `rpm -i /root/rpmbuild/RPMS/aarch64/test-1.0-1.aarch64.rpm --force`
4. `lsmod | grep brd`

According to the preceding steps, the following output can be obtained:
```
[root@localhost SPECS]# lsmod | grep brd
brd                    20480  0
```

Is it reasonable that the brd module is automatically loaded when the block file is installed?

**Comment by JetXujing** [comment]
The brd module is automatically loaded after fsmOpenat() the block file.
```
	    if (!rc && fd == -1 && !S_ISLNK(fp->sb.st_mode)) {
		/* Only follow safe symlinks, and never on temporary files */
		fd = fsmOpenat(di.dirfd, fp->fpath,
				fp->suffix ? AT_SYMLINK_NOFOLLOW : 0, 0);
```

**Comment by pmatilai** [comment]
This seems similar to the FIFO case, the issue being opening special files is special but then we rely on having an open fd to set permissions.

**Comment by pmatilai** [comment]
Hmm, was supposed to be closed automatically by 28c92fd54c93371c3062664d8a938438a2be88d6 but apparently GH doesn't process "Fixes: a, b, c" style lists.
